.all-contributorsrc

```diff
@@ -68,7 +68,17 @@
         "code",
         "doc"
       ]
-    }
+    },
+    {
+      "login": "f3ltron",
+      "name": "Giraud Florent",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/11556276?s=460&v=4",
```

Comment:
```suggestion
      "avatar_url": "https://avatars1.githubusercontent.com/u/11556276?v=4",
```

Reply:
You'll also need to regenerate :) We should probabaly add a bot for this.

Reply:
what u mean by regenerate ?